integration/spark/src/main/scala/org/apache/spark/sql/hive/CarbonFileMetastore.scala

```diff
@@ -430,9 +430,8 @@ class CarbonFileMetastore extends CarbonMetaStore {
     thriftWriter.open(FileWriteOperation.OVERWRITE)
     thriftWriter.write(thriftTableInfo)
     thriftWriter.close()
-    val modifiedTime = System.currentTimeMillis()
-    FileFactory.getCarbonFile(schemaFilePath).setLastModifiedTime(modifiedTime)
-    updateSchemasUpdatedTime(identifier.getCarbonTableIdentifier.getTableId, modifiedTime)
+    updateSchemasUpdatedTime(identifier.getCarbonTableIdentifier.getTableId,
+      System.currentTimeMillis())
```

Comment:
I think, modified time should be the lastModifiedTime of the schema file

Reply:
the setLastModifiedTime funciton in CarbonFileMetastore.scala  is related to other issue:
when delete external table, data loss may happen. I will handle this in another PR.